src/features/landing/components/Header/Buttons/Buttons.tsx

```diff
@@ -0,0 +1,43 @@
+'use client';
+
+import { Button, Stack } from '@mui/material';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+function Buttons() {
```

Comment:
for what reason need this component? this name did not tell me anything about what this component is doing